packages/video-intelligence/src/v1beta1/index.js

```diff
@@ -27,8 +27,8 @@ function v1beta1(options) {
   return videoIntelligenceServiceClient(gaxGrpc);
 }
 
-v1beta1.GAPIC_VERSION = '0.7.1';
+v1beta1.GAPIC_VERSION = '0.0.5';
```

Comment:
I'm just flagging this, because I've seen other people flag it before. Is this correct?